Update staffing change guidance (#254)
18F/identity-handbook: _articles/staffing.md

Follow these steps to re-staff a Login.gov employee internally.

> 1. Reassignment is requested by a Login.gov employee or supervisor.

Reviewer: ```suggestion
1. Reassignment is requested by Login.gov staff, product owner and/or supervisor.
```

> 1. Reassignment is requested by Login.gov staff, product owner and/or supervisor.

Reviewer 2: **comment:** this feels a little... unstructured... to me. I understand not wanting to be too specific about the circumstances but could we maybe have a Google form or something that gets filled out to kick this process off? Something that covers the basics in a consistent way (who the person is, who's making the request, why they're making the request, what the current team is, what the proposed team is, etc). What do you think?

Author: I think I like the unstructured nature, a GForm feels like a step too far; but that is only my opinion.  Curious how others feel too.  I added some of the info you're looking for as sub-bullets, does that help?

Reviewer 2: Fair enough, works for me (and if we run into issues we can adjust)!

1. Create a [Leads Trello Card](https://trello.com/b/C3fvwWz0/login-leadership-master) for the staffing change to hold notes including feedback summaries, effective dates, etc.

> 1. Gather feedback about the change

Reviewer 2: **comment:** it would be great to have someone independent specifically responsible for managing this process (e.g. Clint or a delegate) who would facilitate and perhaps even be present for all discussions to ensure consistency/equity.

Author: I don't think adding names here is the right move, but roles could make sense.  I took a shot, does this help?

Reviewer 2: Yup I think so - one comment is that I don't know if the ED should be involved in personnel decisions at this level but that's pretty nitpick-y. I'm good with this as-is!

>     1. A supervisor or lead has a conversation with the human that is potentially changing teams to get their feedback.

Reviewer: ```suggestion
    1. A Login.gov Supervisor and/or Lead has a conversation with the staff member that is potentially changing teams to get a better understanding of their perspective.
```

>     1. On both teams, a supervisor or lead discusses with the Scrum Master and gets feedback.

Reviewer: ```suggestion
    1. On both teams, a Login.gov Supervisor and/or Lead discusses with the assigned Product Owner and Scrum Master for those teams on plans for this change.
```

    1. On both teams, the Scrum Master informs the team, ex: during stand-up. 
1. This staffing change is included as a topic in weekly staffing meetings. Once all required conversations have occurred and there is no blocking feedback, the employee switches teams. If there is blocking feedback, the employee does not switch teams.

> 1. A decision is documented in the Trello card and it moves to Done.

Reviewer: ```suggestion
1. The staffing decision is documented as completed in the assigned Trello card and it moves to Done.
```
